sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/RecognizeEntitiesOptions.java

```diff
@@ -37,4 +41,26 @@ public RecognizeEntitiesOptions setIncludeStatistics(boolean includeStatistics)
         super.setIncludeStatistics(includeStatistics);
         return this;
     }
+
+    /**
+     * Get the value of {@code stringIndexType}.
+     *
+     * @return The value of {@code stringIndexType}.
+     */
+    public StringIndexType getStringIndexType() {
```

Comment:
we actually added this property to the `TextAnalyticsRequestOptions`. although it doesn't apply to all endpoints, they won't fail if u add it. it is generic enough that if the service for whatever reason adds the property everywhere, it will work

Reply:
^ this work could be done for next sprint